woocommerce/woocommerce-android
# [Login] remove the prologue survey

### Description
As discussed internally p1667902271165339-slack-C03L1NF1EA3, this PR removes the prologue survey and all of its references.

I also updated the tests for the FirebaseRemoteConfigRepository to not depend on a specific key.

### Testing instructions
1. Clear the app's data.
2. Launch the app.
3. Confirm the app opens to the prologue carousel, and works as expected.
